Makefile

```diff
@@ -1,10 +1,13 @@
 include dev.env
 export
 
+check:
+	pre-commit run
+
 install_deps:
```

Comment:
pre-commit run only runs against already `git add`-ed files, I found it helpful to use this makefile snippet to run it against all changed files in the working dir:

```Makefile
CHANGED_FILES := $(shell git ls-files --modified --other --exclude-standard)
CHANGED_FILES_IN_BRANCH := $(shell git diff --name-only $(shell git merge-base origin/main HEAD))

# tools
.PHONY: pre-commit
pre-commit:  ## check modified and added files (compared to last commit!) with pre-commit
	pre-commit run --files $(CHANGED_FILES)

.PHONY: pre-commit
pre-commit-in-branch:  ## check changed since origin/main files with pre-commit
	pre-commit run --files $(CHANGED_FILES_IN_BRANCH)

.PHONY: pre-commit-all
pre-commit-all: format  ## Check all files in working directory with pre-commit
	pre-commit run --all-files

.PHONY: help
help:     ## Show this help.
	@egrep -h '\s##\s' $(MAKEFILE_LIST) | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m  %-30s\033[0m %s\n", $$1, $$2}'
```

Reply:
Great 👍🏻 
I will add this.